styles.css

```diff
@@ -1,31 +1,70 @@
 
 
 
-.main-menu {
-    width: 100%;
-    background: rgb(223, 240, 69);
-    min-height: 20px;
-    padding: 5px;
-    position: fixed;
-    text-align: center;
+
+
+
+  header {
+    background-image: url(https://wallpaperplay.com/walls/full/2/7/d/364648.jpg);
   }
-  .menu {
-    display: flex;
-    justify-content: space-around;
-    font-weight: 700;
-    list-style-type: none;
+
+  h1 { 
+    color: lightgrey;
+    text-align: justify;
+    font-size: 40px;
+    padding: 20px;
+  }
+
+
+  ul {
+    list-style: none;
     margin: 0 auto;
     padding: 0;
+
   }
+    ul li {
+      display: inline-block;
+      
+    }
+     
+      ul li a {
+        color: whitesmoke;
+        text-decoration: none;
+        padding: 20px;
+        display: block;      
+        
+      }
+    
+   
 
-.Project {
-    background-color: gray;
-    padding: 20px;
-}
+  body {
+    
+    background-image: url(https://images.unsplash.com/photo-1592099086579-047cab88633c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=634&q=80);
+    
+  }
 
 .Project-image {
```

Comment:
Just a note, it's considered a convention to use "kabob-case" for CSS class names

(i.e. `.project-image` instead of `.Project-image`)